apps/api/src/cache/cache.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable, Logger, OnModuleDestroy } from '@nestjs/common'
+import { createClient, RedisClientType } from 'redis'
+import { User } from '@prisma/client'
+
+@Injectable()
+export class CacheService implements OnModuleDestroy {
+  private readonly redisClient: RedisClientType
```

Comment:
Hey! I absolutely forgot to mention, we already have redis cache set up. You can directly use it https://github.com/keyshade-xyz/keyshade/blob/develop/apps/api/src/provider/redis.provider.ts.

So, you can skip the initialization step.

Reply:
Done, also updated unit tests to mock redis functions.